classes/classes/Appearance.as

```diff
@@ -2209,6 +2209,7 @@
 					[SKIN_TYPE_UNDEFINED, "undefined flesh"],
 					[SKIN_TYPE_DRAGON_SCALES, "scales"],
 					[SKIN_TYPE_FISH_SCALES, "scales"],
+					[SKIN_TYPE_FLEECE, "fleece"]
```

Comment:
I'd suggest, that you leave a trailing comma in the last line here. So future diffs are smaller. Its not mandatory, but it's common practice.

Same in the changes below.